src/snowflake/cli/plugins/git/manager.py

```diff
@@ -27,3 +30,21 @@ def create(
         if secret is not None:
             query += f"git_credentials = {secret}\n"
         return self._execute_query(query)
+
+    def execute(
+        self,
+        repository_path: str,
+        on_error: OnErrorType,
+        variables: Optional[List[str]] = None,
+    ):
+        return super().execute(
+            stage_path=repository_path, on_error=on_error, variables=variables
+        )
```

Comment:
How different is this from parent's execute?

Reply:
It isn't different :/ . Can be removed